bundles/ch.elexis.core.ui.medication/src/ch/elexis/core/ui/medication/views/MedicationComposite.java

```diff
@@ -784,6 +819,22 @@ public void updateUi(IPatient patient, boolean forceUpdate) {
 			lblDailyTherapyCost.setText(StringUtils.EMPTY);
 			interactionLink.updateAtcs(Collections.emptyList());
 		}
+		if (patient != null && !patient.getId().isEmpty()) {
+			Object extInfoObj = patient.getExtInfo("lastModifiedInfo");
+			String lastModifiedInfo = extInfoObj != null ? extInfoObj.toString() : "";
+			if (lastModifiedInfo != null && !lastModifiedInfo.isEmpty()) {
+				String[] parts = lastModifiedInfo.split(" ");
```

Comment:
Hier fehlt der check ob parts.length >= 2 ist. Sonst gibt das hier eine exception.

Ausserdem wenn der Name einen " " beinhalten würde wäre in parts[1] nicht das Datum.